app/Http/Controllers/TournamentAdminController.php

```diff
@@ -0,0 +1,150 @@
+     * hierin bevinden zich de gebruikers van het toernooi met hun bijbehorende rol(len).
+     * @param $tournamentId
+     * @return void
+     */
+    public function show($tournamentId)
+    {
+        // TODO: Check inbouwen om te kijken of de huidige gebruiker wel een beheerder is van het toernooi.
+        // Hiervoor middleware gebruiken.
+
+        $tournament = Tournament::findOrFail($tournamentId);
+
+        // Alle tournamentUserRole records voor dit toernooi.
+        $tournamentUserRoles = TournamentUserRole::where('tournament_id', $tournament->id)->get();
+        // Alle gebruikers van het toernooi (inclusief beheerders, deelnemers etc).
+        $tournamentUsers = $tournamentUserRoles->unique('user_id')->map(function ($tournamentUserRole) use ($tournamentUserRoles) {
+            // Vind de gebruiker waarover geitereerd wordt.
+            $tournamentUser = User::find($tournamentUserRole->user_id);
+            // Vraag de rollen op voor deze gebruiker uit de niet unieke collectie.
+            $userRoles = $tournamentUserRoles->where('user_id', $tournamentUserRole->user_id)
+                ->map(function ($user) {
+                    // Return de gevonden role voor deze gebruiker.
+                    return Role::find($user->role_id)->name;
+                })->toArray();
+
+            // TODO: Hardcoded rol namen statisch maken.
+            $tournamentUser->isOrganizer = array_search('organizer', $userRoles) > -1
+                ? true : false;
+            $tournamentUser->isParticipant = array_search('participant', $userRoles) > -1
+                ? true : false;
+
+            return $tournamentUser;
+        });
+
+        return view('tournament.admin.show')->with([
+            'tournament' => $tournament,
+            'tournamentUsers' => $tournamentUsers
+        ]);
+    }
+
+    /**
+     * Verwijder een gebruiker van een toernooi.
+     * @param $tournamentId
+     * @param $userId
+     * @param $roleName
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function deleteUser($tournamentId, $userId, $roleName)
+    {
+        $role = Role::getByName($roleName);
+
+        if (!$role) {
+            return redirect()->back()
+                ->withErrors([
+                    'De opgegeven rol kon niet gevonden worden.'
+                ]);
+        }
+
+        if ($roleName === 'organizer') {
+            $totalTournamentOrganizerCount = TournamentUserRole::where([
+                'tournament_id' => $tournamentId,
+                'role_id' => $role->id
+            ])->count();
+
+            if ($totalTournamentOrganizerCount < 2) {
+                return redirect()->back()
+                    ->withErrors([
+                        'De beheerder kan niet verwijderd worden omdat er altijd minimaal één beheerder moet zijn.'
+                    ]);
+            }
+        }
+        // Vind de record in de TournamentUserRole table.
```

Comment:
Nog een check maken om een deelnemer pas te verwijderen wanneer het toernooi nog niet gestart is